dependencies/pom.xml

```diff
@@ -119,6 +119,8 @@
         <version.lib.mongodb.reactivestreams>1.11.0</version.lib.mongodb.reactivestreams>
         <version.lib.mssql-jdbc>8.4.1.jre8</version.lib.mssql-jdbc>
         <version.lib.neo4j>4.4.3</version.lib.neo4j>
+        <version.lib.hibernate-validator>6.2.1.Final</version.lib.hibernate-validator>
+        <version.lib.el>3.0.0</version.lib.el>
```

Comment:
This doesn't seem to be used anywhere

Reply:
`<version.lib.el>3.0.0</version.lib.el>`  removed